src/CMakeLists.txt

```diff
@@ -37,15 +41,15 @@ endif(NATS_BUILD_STREAMING)
 # --------------------------------------
 if(NATS_BUILD_LIB_SHARED)
   add_library(nats SHARED ${SOURCES} ${GLIB_SOURCES} ${PS_SOURCES} ${S_SOURCES})
-  target_link_libraries(nats ${NATS_OPENSSL_LIBS} ${NATS_EXTRA_LIB} ${NATS_PROTOBUF_LIBRARIES} ${NATS_SODIUM_LIBRARIES})
+  target_link_libraries(nats ${NATS_OPENSSL_LIBS} ${NATS_WOLFSSL_LIBRARIES} ${NATS_EXTRA_LIB} ${NATS_PROTOBUF_LIBRARIES} ${NATS_SODIUM_LIBRARIES})
```

Comment:
If we support different SSL libraries, I think we should have a `NATS_SSL_LIBS` that would be set to either one.